Makefile

```diff
@@ -120,7 +122,9 @@ BINARIES=$(addprefix bin/,$(COMMANDS))
 TESTFLAGS ?= $(TESTFLAGS_RACE) $(EXTRA_TESTFLAGS)
 TESTFLAGS_PARALLEL ?= 8
 
-.PHONY: clean all AUTHORS build binaries test integration generate protos checkprotos coverage ci check help install uninstall vendor release mandir install-man genman
+CRIDIR=$(ROOTDIR)_output/cri
```

Comment:
$(ROOTDIR) has a `/` at the end?

Reply:
it does, updated to use `join` to create the output dir